### DISCUSSION TITLE
Particle Mass Conservation

### DISCUSSION THREAD

**Discussion by amarcozzi** [firemodels/fds#12722] [Q&A] [answered]
Hello, I am working on a set of FDS cases where I am looking at fire behavior as a function of fuel bed depth with vegetative fuel similar to that found in the CSIRO grassland case. In those simulations I input particles in the domain with measured mass per volume and devices to measure the spatially integrated mass per volume. The numbers output by the device match what I expect the mass per volume to be.

However, I noticed that when I sum the PARTICLE MASS output quantity for all of the particles in a control volume, then I get a total mass that is significantly lower than what I expect to see. I ran a couple of simplified 1m x 1m x 1m domains with varying fuel heights and I continue to notice a difference between the input mass per volume and the sum of the output particle mass.

To demonstrate this problem I ran the ['pine_needles' test case](https://github.com/firemodels/fds/blob/master/Verification/WUI/pine_needles.fds) with the FDS 6.9.0 release version. This case places 1kg of wet pine needles in a 1m cube with 1000 particles. If we look at the devc.csv file, then it looks like the solid mass quantity is 1kg, which is what we expect, and that is also reflected in Figure 17.4 of the user guide.

When I look at the particle mass output quantity, then I see something strange. Given that there are 1000 particles to represent 1kg of pine needles, I would expect each particle to have a mass somewhere in the neighborhood of 1g for a total of 1kg. However, it looks like each particle has a mass of approximately 0.012g for a sum total of approximately 12g. So, according to this method of looking at the total mass of vegetation there are 988g missing. I've attached a smokeview screenshot below showing the distribution of particles colored by the PARTICLE MASS quantity, as well as a plot of sum total particle mass over time.

Am I thinking about what a particle represents incorrectly, or misunderstanding the PARTICLE MASS output quantity? If not, is FDS distributing less than the input mass to particles throughout the domain?

I ran the pine_needles.fds case exactly as it is in the Verifaction/WUI directory so I haven't included my input or output files, but please let me know if there is anything else that I can provide.

Thanks,
Anthony

<img width="752" alt="Screenshot 2024-04-01 at 11 58 27 AM" src="https://github.com/firemodels/fds/assets/13575005/e9a46b61-9da2-41e0-a9fe-4241dacaab0c">

<img width="697" alt="Screenshot 2024-04-01 at 12 11 44 PM" src="https://github.com/firemodels/fds/assets/13575005/20f81b3e-18cd-4152-9eca-966bb0b4dc19">

**Comment by mcgratta** [comment, marked as answer]
The `PARTICLE MASS` is the actual mass of the particle, not the weighted mass of a collection of particles that it represents.

**Comment by amarcozzi** [comment]
Ah, I see. So it looks like if I multiply PARTICLE MASS by PARTICLE WEIGHTING FACTOR, then that will give me the expected weighted mass of a collection of particles. Am I correct in that interpretation?


<img width="776" alt="Screenshot 2024-04-01 at 12 29 28 PM" src="https://github.com/firemodels/fds/assets/13575005/3bb5e2f1-048f-4194-8f08-d5a435ee28fa">

**Comment by mcgratta** [comment]
Yes

**Comment by amarcozzi** [comment]
Thank you for your help!
